Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:18

```

Comment:
`node:18` is no longer in "active support" according to [endoflife.date](https://endoflife.date/nodejs)

Reply:
well i got `node:18` from this file [package-lock.json](https://github.com/ryohey/signal/blob/119b0e0b5a5b80295fcce34b6085436993e7e7e0/package-lock.json#L123C1-L125C8)

```
...
      },
      "engines": {
        "node": ">=18.0.0"
      }
    },
...
```
which specify the minimum version of node that would be used, and didn't check the EOL of that version, and currently I'm too lazy to check what is the latest version of node that can be use without breaking the project

Reply:
Since it will run in Docker the user probably won't care what version it is as long as it works, so `node:lts` seems to be the happy medium

Reply:
ok yea that sounds far better than manually checking if a version would break this project or not